clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -485,20 +486,48 @@ static void addPGOAndCoverageFlags(const ToolChain &TC, Compilation &C,
   }
 
   if (ProfileUseArg) {
+    SmallString<128> Path;
```

Comment:
Nit: I suggest something like `PathBuf` or `PathStorage` so we don't mix this up with the desired path. I know that increases the diff below slightly, but seems more clear to me.